ledger-bridge.js

```diff
@@ -38,7 +40,16 @@ export default class LedgerBridge {
                         this.cleanUp(replyAction)
                         break
                     case 'ledger-update-transport':
-                        this.updateLedgerLivePreference(replyAction, params.useLedgerLive)
+                        if (params.transportType === 'ledgerLive' || params.useLedgerLive) {
+                            this.updateLedgerLivePreference(replyAction, true)
+                            this.updateWebHidPreference(replyAction, false)
+                        } else if (params.transportType === 'webhid') {
+                            this.updateLedgerLivePreference(replyAction, false)
+                            this.updateWebHidPreference(replyAction, true)
```

Comment:
We're now sending two replies to a single message here, which is not how this is supposed to work.

Rather than calling both of these functions, maybe we can have a single "updateTransport" function? And use an enum in place of `this.useWebHid` and `this.useLedgerLive`. That would ensure that only one transport is enabled at a time, and that we only reply once to this message.